server/room-battle.ts

```diff
@@ -1203,6 +1217,12 @@ export class RoomBattleStream extends BattleStream {
 			this.battle.inputLog.push(`>chat ${message}`);
 			this.battle.add('chat', `${message}`);
 			break;
+		case 'reseed':
```

Comment:
Putting this in `RoomBattleStream` instead of `BattleStream` makes it incompatible with `./pokemon-showdown simulate-battle`, which is bad.